src/serializers/mod.rs

```diff
@@ -16,7 +16,7 @@ pub use shared::CombinedSerializer;
 use shared::{to_json_bytes, BuildSerializer, TypeSerializer};
 
 mod computed_fields;
-mod config;
+pub(crate) mod config;
```

Comment:
Why this change?

Reply:
I guess my thought of a `common` module would solve this https://github.com/pydantic/pydantic-core/pull/1308#discussion_r1697207291